applications/RELAI_Network.md

```diff
@@ -0,0 +1,146 @@
+
+Org:
+
+https://github.com/RELAI-Network
+
+Team:
+
+https://github.com/morgueye4
+
+## Development Status :open_book:
+
+To this date, researches on the subject of Blockchain and AI intersection have been started and other researches are ongoing for the next phases. The phase 1 implementation is ready to be implemented.
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** 2 months
+- **Full-Time Equivalent (FTE):** 1 FTE
+- **Total Costs:** 10,000 USDC
+
+
+### Milestone 1 — Implementing the Blockchain Layer
+
+- **Estimated Duration:** 1 month
+- **FTE:**  1
+- **Costs:** 5,000 USDC
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | Apache 2.0 |
+| 0b. | Documentation | We will provide both **inline documentation** of the code and a **tutorial** that explains how a user can setup RELAI node and interact with the Pallets.|
+| 0c. | Testing Guide | Core functions will be fully covered by unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| 0d. | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 1. | `MLOrg` Pallet | This pallet will be responsible for creating and managing Organizations and access for the users  `Create organization`, `Add/Remove users`, `Assign roles` (eg: Admin, AI engineer, MLOps, Stakeholders,…)|
+| 2. | `ModelBase` Pallet | The ModelBase pallet will be responsible for managing models and will allow to: `Create/Update` project ,`Create/Update` model artifact ,`Remove model` artifact ,`Create/Update` model version, `Create/Update` model experiment ,`Create/Update` model review. There will be data structures for the : `Projects`, `ProjectPurpose`, `Model`, `ModelVersion`, `ModelArtifact`, `ModelExperiment`, `ModelReview`. ModelBase pallet extrinsics will be called according to the caller's role (managed by the MLOrg pallet which will be loosely coupled).|
+
+
+### Milestone 2 — Implementing the Front Layer
+
+- **Estimated Duration:** 1 month
+- **FTE:**  1
+- **Costs:** 5,000 USDC
+
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | Apache 2.0 |
+| 0b. | Documentation | Documentation of code and  tutorials that explains how to use the Web apps for full AI Models management lifecycle. Benefits of such systems wil also be explained |
+| 0c. | Testing Guide | Core functions will be fully covered by unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| 0d. | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 1. | Web Application | We will create a web frontend application that will be in charge of interacting with the blockchain and other layers such as storage a for model storage and also for Organisations management | 
```

Comment:
Thanks for the application. Can you specify which front-end framework you will use here? Also, can you expand on what functionalities the SDK will provide?